### ISSUE TITLE
Cannot add matrix field to SuperTable field (Craft 4)

### ISSUE THREAD

**Issue by ivarsbariss-solspace** [engram-design/SuperTable#465]
**Description**
When adding a Matrix field to a new SuperTable field, the Matrix field does not allow to create a new block type.  When clicking on "+ New Block Type" there isn't any response and the browser's console does not show any errors.

**Steps to reproduce**
1. Start creating a new SuperTable field
2. Add a new Matrix field to the SuperTable field you are currently creating
3. Try adding a new Block Type to the Matrix Field

**Additional info**
- Plugin version: 3.0.0-beta.3
- Craft version: 4.0.1
- Multi-site: No

<img width="1251" alt="image" src="https://user-images.githubusercontent.com/76444614/168065104-312b7be4-f5f8-4ad4-8d08-94230343a6b1.png">

**Comment by engram-design** [comment]
Duplicate of https://github.com/verbb/super-table/issues/459 Fixed for the next release. To get the fix early, change your `verbb/super-table` requirement in `composer.json` to:

```
"require": {
  "verbb/super-table": "dev-craft-4 as 3.0.0-beta.3",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [3.0.0-beta.4](https://github.com/verbb/super-table/releases/tag/3.0.0-beta.4)
